VERSIONING.md

```diff
@@ -5,10 +5,25 @@ This site uses [Docusaurus' built in support for versioning](https://docusaurus.
 - _/docs/mainnet/_ represents the last deployed version on [mainnet](https://blog.vega.xyz/what-to-expect-from-restricted-mainnet-616086d9fdaf)
 - _/docs/testnet/_ represents the most recent version on [Fairground testnet](https://fairground.wtf)
 
-Both of those do not neccessarily move in lock step. There could be two version updates to one network before the other updates. Bumping a testnet version is easy. Moving a testnet version to mainnet is harder.
 
-# Changing the testnet version
-When a new release is *tagged*, a pull request with the API documents will automatically be create. It will add a new folder in `/specs/`, for example `v0.88.1`. When the network is updated:
+# Overview
+![update-process png excalidraw](https://user-images.githubusercontent.com/6678/200811843-564a0127-2f25-4f6b-ae56-10305cf1ddfa.png)
+
+Every time a release is [tagged in vegaprotocol/vega](https://github.com/vegaprotocol/vega), a [pull request is created in this repository](https://github.com/vegaprotocol/documentation/pulls?q=is%3Apr+automated+specs+update).  
+
+1. When the release is adopted on testnet, in that branch, follow the [Changing the testnet version](#changing-the-testnet-version) instructions
+2. Patch releases that are deployed to the network follow the same update process as above. Patches that do not get deployed to testnet don't require an update - the pull request can be closed.
+3. When a release is adopted on mainnet, follow the [Changing the mainnet version](#changing-the-mainnet-version) instructions.
+
+--- 
+
+## tl;dr
+- [/docs/](https://github.com/vegaprotocol/documentation/tree/main/docs/) is 'the current testnet version'. Every time testnet is updated, the docs here will be updated. This folder is served as `/testnet/`.
+- [/versioned_docs/](https://github.com/vegaprotocol/documentation/tree/main/versioned_docs/) contains version folders. The most recent version in this folder is served as `/mainnet/`.
+- Only one version is kept in `versioned_docs`. [vegaprotocol/vega](https://github.com/vegaprotocol/vega). This is because each version contains a lot of pages, which slows down the build - and at this point in time, the benefit of keeping old versions is small.
+
+## Changing the testnet version
+When a new release is *tagged*, a pull request with the API documents will automatically be create. It will add a new folder in `/specs/`, for example `v0.88.1`. Only those versions that actually get released need to be merged, so **when the network is updated**:
```

Comment:
```suggestion
When a new release is *tagged*, a pull request with the API documents will automatically be created. It will add a new folder in `/specs/`, for example `v0.88.1`. Only those versions that actually get released need to be merged, so **when the network is updated**:
```